scripts/firefox-build-manifest.js

```diff
@@ -0,0 +1,52 @@
+const { exec } = require("child_process");
+const fs = require("fs");
+
+const RELATIVE_PATH_TO_MANIFEST = "./dist/manifest.json";
```

Comment:
As the script was moved, this should be updated:

```suggestion
const RELATIVE_PATH_TO_MANIFEST = "../dist/manifest.json";
```